src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -327,6 +327,10 @@ public void Dispose()
                 // Esc[?1h - Set cursor key to application mode
                 // See http://ascii-table.com/ansi-escape-sequences-vt-100.php
                 Console.Write("\u001b[?1h");
+
+                // Similar to the fix above this, this turns of Mouse Tracking as described here:
+                // https://www.xfree86.org/current/ctlseqs.html#Mouse%20Tracking
+                Console.Write("\u001b[?1003l");
```

Comment:
Any particular reason to do it in two packets instead of one?

Reply:
good point. Combined them into one `Console.Write`